wgsl/index.bs

```diff
@@ -4505,6 +4525,10 @@ There are three kinds of constructor expressions:
 * [[#zero-value-expr]]
 * [[#conversion-expr]]
 
+In the following sections, when a type name precedes a parenthesized argument list, any user-defined or [=predeclared=]
+[=type alias|alias=] for that type can be used instead, with the same effect.
```

Comment:
```suggestion
In the following sections, when a type name precedes a parenthesized argument list, any 
[=type alias|alias=] for that type can be used instead, with the same effect.
```
You could add the user-defined or predeclared back as a note, but I feel the clarification causes the user to ask if that is really a subset or not.
> Note: [=type alias|Types aliases=] may be either [=predeclared=] or user-defined.

Reply:
Good suggestions.  Added.